src/locales/en.json

```diff
@@ -1,13 +1,16 @@
 {
   "common": {
     "additional_services": "Additional services",
+    "berth": "Berth",
     "cancelled": "Cancelled",
     "dinghy_place": "Dinghy place",
+    "drafted": "Drafted",
     "due_date": "Due date",
     "electricity": "Electricity",
     "error": "Error",
     "expired": "Expired",
     "gate": "Gate",
+    "harbor": "Harbour",
```

Comment:
Where's this coming from? If we want to use "harbour", there's at least the line "Harbor map (PDF)" that also needs to be changed

Reply:
I don't know why it is initially "harbour" and not "harbor" on the customer UI, I changed it here for the sake of consistency.
I'll change "Harbor map (PDF)" as well.

Reply:
"Fixed" the remaining translations but pushed to the other PR 😅   https://github.com/City-of-Helsinki/berth-reservations-ui/pull/390